app/helpers/date_helper.rb

```diff
@@ -56,6 +56,14 @@ def human_time(dt)
     end
   end
 
+  def display_as_time(minutes)
```

Comment:
I don't have a problem with adding a view helper method for it, but we have `MinutesToTimeFormatter` which includes this logic.

Reply:
Ah thank you, I thought we had something like that but couldn't find it.